src/main/fc/cli.c

```diff
@@ -890,6 +892,20 @@ static const clivalue_t valueTable[] = {
     {"ltm_update_rate",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_LTM_UPDATE_RATE }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, ltmUpdateRate) },
 #endif
 #endif
+#ifdef USE_RX_ELERES
+//PG_ELERES_CONFIG
+    { "eleres_freq",                VAR_FLOAT | MASTER_VALUE, .config.minmax = { 415, 450 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresFreq) },
+    { "eleres_telemetry_en",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresTelemetryEn) },
+    { "eleres_telemetry_power",     VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 7 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresTelemetryPower) },
+    { "eleres_loc_en",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresLocEn) },
+    { "eleres_loc_power",           VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 7 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresLocPower) },
+    { "eleres_loc_delay",           VAR_UINT16 | MASTER_VALUE, .config.minmax = { 30, 1800 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresLocDelay) },
+    { "eleres_signature_1",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresSignature[0]) },
+    { "eleres_signature_2",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresSignature[1]) },
+    { "eleres_signature_3",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresSignature[2]) },
+    { "eleres_signature_4",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_ELERES_CONFIG, offsetof(eleresConfig_t, eleresSignature[3]) },
```

Comment:
Because signature is written to registers byte, by byte, and simple trick like this is not working (probably because of PG_xxxx macros etc.):

`(uint8_t*) signature = &eleresConfigMutable()->eleresSignature;
or 
uint8_t signature[4];
memcpy(signature ,&eleresConfigMutable()->eleresSignature, 4);`